src/watson/urls.py

```diff
@@ -2,13 +2,13 @@
 
 from __future__ import unicode_literals
 
-from django.conf.urls import url, patterns
+from django.conf.urls import url
+from . import views
```

Comment:
Relative imports can lead to Python modules being imported twice, as they
can resolve to different names. This is avoided by using absolute imports.

It's often harmless, as most module imports don't have side effects, but it
can lead to things like registration code running twice, which can cause
confusing errors.
On Tue, 19 Jan 2016 at 11:51, Eric Carmichael <notifications@github.com>
wrote:

> In src/watson/urls.py
> <https://github.com/etianen/django-watson/pull/148#discussion_r50103065>:
>
> > @@ -2,13 +2,13 @@
> >
> >  from __future__ import unicode_literals
> >
> > -from django.conf.urls import url, patterns
> > +from django.conf.urls import url
> > +from . import views
>
> Heyo thanks for all the work! I'm curious about this, why no relative
> imports? Seems to me to be the "best practice" way of doing things?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/etianen/django-watson/pull/148/files#r50103065>.
>